src/DDTrace/Util/CodeTracer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace DDTrace\Util;
+
+use DDTrace\GlobalTracer;
+
+class CodeTracer
```

Comment:
Just to lock this down a bit. :)
```suggestion
final class CodeTracer
```